scripts/absolute_velocity_from_switches.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import sys
+import rospy
+
+from iarc7_msgs.msg import LandingGearContactsStamped
+from geometry_msgs.msg import TwistWithCovarianceStamped
+
+from iarc7_safety.SafetyClient import SafetyClient
+
+def callback(msg):
+    if msg.front and msg.back and msg.left and msg.right:
+        # We're on the ground
+        out_msg = TwistWithCovarianceStamped()
+        out_msg.header.frame_id = 'map'
+        out_msg.header.stamp = msg.header.stamp
+        pub.publish(out_msg)
+    global last_msg_stamp
+    last_msg_stamp = msg.header.stamp
+
+last_msg_stamp = None
+
+if __name__ == '__main__':
+    rospy.init_node('absolute_vel_from_switches')
+    rospy.Subscriber('landing_gear_contact_switches', LandingGearContactsStamped, callback)
+    pub = rospy.Publisher('absolute_vel', TwistWithCovarianceStamped, queue_size=0)
+
+    # Wait for a valid timestamp
+    while rospy.Time.now() == rospy.Time(0):
+        if rospy.is_shutdown():
+            sys.exit()  
```

Comment:
Can we move all this code into a `main` function, and have this return instead of calling `sys.exit`?  `sys.exit` is almost never the right thing to do